### PR TITLE
Added some tests to validate that Custom converters on wrapper primitive types are applied to both the wrapper type and the primitive type.

### PR DESCRIPTION
**Pull request by radcortez**

### CODE REVIEW

**Comment by radcortez** [comment]
TCK tests for https://github.com/eclipse/microprofile-config/pull/518

**Comment by genie-microprofile** [comment]
Can one of the admins verify this patch?

**Comment by radcortez** [comment]
I'll add them.

**Comment by radcortez** [comment]
Done.

File: tck/src/main/resources/internal/META-INF/microprofile-config.properties [hunk 1]
```diff
@@ -138,6 +138,11 @@ tck.config.test.javaconfig.converter.implicit.sequence.ofBeforeValueOf=ofBeforeV
 tck.config.test.javaconfig.converter.implicit.sequence.valueOfBeforeParse=valueOfBeforeParse
 tck.config.test.javaconfig.converter.implicit.sequence.parseBeforeConstructor=parseBeforeConstructor
 
+# custom converters
+tck.config.test.javaconfig.custom.converter.integervalue=1
+tck.config.test.javaconfig.custom.converter.longvalue=1
+tck.config.test.javaconfig.custom.converter.doublevalue=1.0
+tck.config.test.javaconfig.custom.converter.booleanvalue=false
```
Comment (Emily-Jiang) [inline tck/src/main/resources/internal/META-INF/microprofile-config.properties]: @radcortez can you also add a charvalue here to ensure this value is not used.

Comment (radcortez) [reply tck/src/main/resources/internal/META-INF/microprofile-config.properties]: Done!

**Pull request merged**